### PR TITLE
Redesign reload indicator and add notification

### PR DESCRIPTION
**Pull request by aumayr**
PR for #405

### CODE REVIEW

File: fava/static/javascript/main.js [hunk 1]
```diff
@@ -91,6 +91,10 @@ e.on('info', (msg) => {
   $('#notifications').insertAdjacentHTML('beforeend', `<li>${msg}</li>`);
 });
 
+e.on('warning', (msg) => {
+  $('#notifications').insertAdjacentHTML('beforeend', `<li class="warning">${msg}</li>`);
```
Comment (aumayr) [inline fava/static/javascript/main.js]: @yagebu How do I add additional args to the `trigger(...)`-call? And how should the auto-dismiss be implemented?

File: fava/static/javascript/main.js [hunk 2]
```diff
@@ -102,6 +106,7 @@ function doPoll() {
       if (data.changed) {
         $('#reload-page').classList.remove('hidden');
         e.trigger('file-modified');
+        e.trigger('warning', _('Beancount file changed outside of Fava. Reload is required.'));
```
Comment (yagebu) [inline fava/static/javascript/main.js]: The first sentence is not entirely correct e.g. a document could have been added. Also, "required" sounds a bit harsh ;) Maybe (the "click to reload" would be nice IMHO):
> File change detected. Click to reload.

**Comment by aumayr** [comment]
@fokusov Can you help with the Russian translation on this? Just this string is missing:

```
    "File change detected. Click to reload."
```

Can you just post the translation as a comment here? Thanks!

**Comment by fokusov** [comment]
`Can you just post the translation as a comment here? Thanks!`

"Файл изменён извне. Нажмите, чтобы перезагрузить."

**Comment by aumayr** [comment]
Thanks @fokusov!

**Pull request merged**